### PR TITLE
add zip to Docker image; add Structured-Streaming to list of Self-Paced courses

### PR DESCRIPTION
**Pull request by joshuacook**

### CODE REVIEW

**Comment by joshuacook** [comment]
Note that this PR also updated the name of `courses/self-paced` to `courses/Self-Paced`. This is part of the repo reorg PR. This should be merged after that PR is merged.

File: course [hunk 1]
```diff
@@ -29,9 +29,9 @@ function configure() {
   fi
 
   # Navigation logic for self-paced courses
-  SELF_PACED_COURSES="Spark-SQL DataFrames ETL-Part-1 ETL-Part-2 ETL-Part-3 Delta"
+  SELF_PACED_COURSES="Structured-Streaming Spark-SQL DataFrames ETL-Part-1 ETL-Part-2 ETL-Part-3 Delta"
   if  [[ "${SELF_PACED_COURSES}" =~ (^|[[:space:]])"${COURSE_NAME}"($|[[:space:]]) ]]; then
-    PREFIX="${PREFIX:-"self-paced/"}"
+    PREFIX="${PREFIX:-"Self-Paced/"}"
```
Comment (dougbateman) [inline course]: Are you sure about this capitalization change?  I believe the directory name is git is lower case.
Ultimately what I really want to do is have a "search path" where it looks for a course in /courses and then in /courses/self-paced.  I just haven't gotten to code that.

Comment (joshuacook) [reply course]: This was part of the move I just did.

Comment (dougbateman) [reply course]: Ah, I see it now.  Roger that.  Merging.

**Pull request merged**